pyramid_storage/gcloud.py

```diff
@@ -241,6 +247,15 @@ def save_file(self, file, filename, folder=None, bucket_name=None, randomize=Fal
 
         blob.cache_control = self.cache_control
         file.seek(0)
-        acl = acl or self.acl
-        blob.upload_from_file(file, rewind=True, content_type=content_type, predefined_acl=acl)
+
+        kwargs = {
+            "rewind": True,
+            "content_type": content_type or "application/octet-stream",
+        }
+
+        if not self.uniform_bucket_level_access:
+            kwargs["predefined_acl"] = acl or self.acl
```

Comment:
Could you add a `else` here that logs a warning if `acl` or `self.acl` is not `None`, to avoid silently ignoring acls?